guides/Getting Started/GettingStarted.mdx

```diff
@@ -0,0 +1,56 @@
+import { saphFrameworkUrl, djsUrl, createDjsTypedefUrl } from '../constants.mdx';
+
+export const djsClientOptions = createDjsTypedefUrl('ClientOptions');
+export const name = '@sapphire/framework';
+
+# Welcome to the @sapphire/framework#{@branch} documentation
+
+## Installing <a href={saphFrameworkUrl}>@sapphire/framework</a>
+
+You can install this version of @sapphire/framework with one of the following commands:
+
+**When using `npm`**
+
+```sh
+npm install @sapphire/framework
+```
+
+**When using `yarn`**
+
+```sh
+yarn add @sapphire/framework
+```
+
+**When using `pnpm`**
+
+```sh
+pnpm add @sapphire/framework
+```
+
+### Using @sapphire/framework
+
+Create a file called `app.js` (or whatever you prefer) which will initiate and configure @sapphire/framework.
+
+```javascript
+const { SapphireClient } = require('@sapphire/framework');
+
+const client = new SapphireClient();
+
+client.login('your-bot-token');
+```
+
+### Client Options: {@link SapphireClientOptions}
+
+{@typedef SapphireClientOptions}
```

Comment:
```suggestion
{@typedoc SapphireClientOptions}
```
According to the above documentation. If this isn't incorrect, L27 of CONTRIBUTING.md is.